samples/rasterization/src/rasterization.zig

```diff
@@ -341,11 +334,6 @@ fn init(allocator: std.mem.Allocator) !DemoState {
         ) catch |err| hrPanic(err),
     };
 
-    for (mesh_textures) |texture| {
```

Comment:
pretty sure this is deadcode that all it did was take up memory

Reply:
@michal-z can you confirm if it's ok to remove this?